products/motorola.md

```diff
@@ -0,0 +1,429 @@
+  support: true
+  eol: 2025-05-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11247
+
+- releaseCycle: "motorola edge+ (2022)"
+  releaseDate: 2022-02-01
+  support: true
+  eol: 2025-02-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11221
+
+- releaseCycle: "motorola edge (2022)"
+  releaseDate: 2022-08-01
+  support: true
+  eol: 2025-08-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11319
+
+- releaseCycle: "motorola edge 20 lite"
+  releaseDate: 2021-08-01
+  support: true
+  eol: 2023-07-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11166
+
+- releaseCycle: "motorola edge 20 fusion"
+  releaseDate: 2021-08-01
+  support: true
+  eol: 2023-08-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11166
+
+- releaseCycle: "motorola edge 20"
+  releaseDate: 2021-08-01
+  support: true
+  eol: 2023-08-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11167
+
+- releaseCycle: "motorola edge (2021)"
+  releaseDate: 2021-08-01
+  support: true
+  eol: 2023-08-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11170
+
+- releaseCycle: "motorola razr 40 ultra"
+  releaseDate: 2023-06-01
+  support: true
+  eol: 2027-06-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11577
+
+- releaseCycle: "motorola razr 40s"
+  releaseDate: 2023-11-01
+  support: true
+  eol: 2027-11-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11651
+
+- releaseCycle: "motorola razr 40"
+  releaseDate: 2023-07-01
+  support: true
+  eol: 2027-07-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11591
+
+- releaseCycle: "motorola razr+ (2023)"
+  releaseDate: 2023-06-01
+  support: true
+  eol: 2027-06-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11489
+
+- releaseCycle: "motorola razr (2023)"
+  releaseDate: 2023-09-01
+  support: true
+  eol: 2027-09-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11592
+
+- releaseCycle: "motorola razr (2022)"
+  releaseDate: 2022-10-01
+  support: true
+  eol: 2025-10-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11371
+
+- releaseCycle: "ThinkPhone by motorola"
+  releaseDate: 2023-01-01
+  support: true
+  eol: 2027-01-01
+  link: https://en-us.support.motorola.com/app/software-security-update/g_id/7112/productid/11418
+
+
+---
+
+> [Motorola Mobility](https://motorola.com) is an American smartphone
+> manufacturer, known for the Moto and Razr brands.
+> It is a subsidiary of Lenovo.
+
+## Security updates
+
```

Comment:
```suggestion
```

No need for a title IMO (see https://deploy-preview-4328--endoflife-date.netlify.app/motorola for the preview).

Reply:
:shrug: I was cribbing from the samsung-mobile page... either way